website/docs/r/log_search.html.markdown

```diff
@@ -10,13 +10,13 @@ Provides a Sumologic Log Search.
 
 ## Example Usage
 ```hcl
-data "sumologic_personal_folder" "personalFolder" {}
+data "sumologic_personal_folder" "personal_folder" {}
 
 resource "sumologic_log_search" "example_log_search" {
     name = "Demo Search"
     description = "Demo search description"
-    parent_id = data.sumologic_personal_folder.personalFolder.id
-    query_string = "_sourceCategory=api error | count by _sourceHost"
+    parent_id = data.sumologic_personal_folder.personal_folder.id
+    query_string = "_sourceCategory=api | parse \"parameter1=*,\" as parameter1 | parse \"parameter2=*,\" as parameter2 | where parameter1 matches {{param1}} | where parameter2 matches {{param2}} | count by _sourceHost"
```

Comment:
This is a nitpick, but for longer query strings I think this is more readable:
```
    query_string = <<QUERY
_sourceCategory=api
| parse \"parameter1=*,\" as parameter1
| parse \"parameter2=*,\" as parameter2
| where parameter1 matches {{param1}}
| where parameter2 matches {{param2}}
| count by _sourceHost
QUERY
```

Reply:
done